example/WebApplication/appsettings.json

```diff
@@ -3,6 +3,8 @@
     "PathFormat": "Logs/log-{Date}.txt",
     "OutputTemplate": "{Timestamp:o} {RequestId,13} [{Level:u3}] {Message} ({EventId:x8}) {Properties:j}{NewLine}{Exception}",
     "IncludeScopes": true,
+    "RollOnFileSizeLimit": true,
+    "FileSizeLimitBytes": 10485760,
```

Comment:
No need to include these here; while they're useful, only a handful of users will want to configure them.

Reply:
yeah just tested locally and left for tests